Zoo.py

```diff
@@ -0,0 +1,100 @@
+# This is a sample Python script.
+import datetime
+
+
+
+class Animal:
+    was_born = False #проверяем было ли рождено животное после инициализации нельзя менять день рождения и цвет
+    birthday = str #Установка дня рождения
```

Comment:
Ты приравнял имя к типу данных str, это не имеет смысла
```suggestion
    birthday: str
```